README.md

```diff
@@ -177,29 +177,14 @@ These aliases are also included in `jsconfig.json` which makes VS Code resolve t
 
 `js/input-detection.js` is included in both the `client` and `static` bundles. It checks for mouse, touch and keyboard events and puts classnames on `<html>`:
 
-* `.mouse-user`: The last event was either a mouse or touch event (this is removed when a keyboard event occurs)
-* `.touchevents`: The last event was a touch event
-* `.no-touchevents`: The last event was not a touch event
+- `.mouse-user`: The last event was either a mouse or touch event (this is removed when a keyboard event occurs)
+- `.touchevents`: The last event was a touch event
+- `.no-touchevents`: The last event was not a touch event
 
 By default, all focus outlines are disabled when the `.mouse-user` class is present (in `site.scss`). No class names are present if js is disabled in the browser, meaning focus outlines are preserved.
 
 You can use these classnames to provide alternative styling based on input method (like disabling hover effects for touch screens).
 
-## UI Testing
```

Comment:
Don't we need any documentation about this at all? I think we at least need to have something about the fact that you need to start webpack dev server in order for tests to work, which isn't obvious.

Reply:
I forgot to add it in again 😞 Will do.